Bottom: ccc0c810e2395e501642fd53f9432ea1045cac64
Top:    20217f47dc32b54ca1aa7c130e025e9bb07a0ff3
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-21 10:55:46 +0300

Refresh of sock-res-completion.patch

---

diff --git a/ib-sock-int.h b/ib-sock-int.h
index c6bcecf..08df596 100644
--- a/ib-sock-int.h
+++ b/ib-sock-int.h
@@ -50,6 +50,8 @@ struct IB_SOCK {
 	/* completion events */
 	struct ib_cq		*is_cq;
 	/* queue pair to communicate between nodes */
+	/* we may create a per CPU QP to reduce a contention
+	 * on single QP */
 	struct ib_qp		*is_qp;
 
 	/* pre-accepted sockets */
